src/main/java/domain/LottoBall.java

```diff
@@ -0,0 +1,55 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class LottoBall implements Comparable<LottoBall> {
+	private final int lottoBall;
```

Comment:
원시값을 포장하는 케이스는 인스턴스 변수의 네이밍이 조금 애매할 때가 있어요. 😃
전 개인적으로 `value`와 같은 네이밍을 추천하는 편이에요. 😉

```suggestion
	private final int value;
```